lore/settings.py

```diff
@@ -219,6 +219,11 @@ def get_var(name, default):
     DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
 
 
+# Lore preview settings
+LORE_PREVIEW_BASE_URL = get_var(
+    'LORE_PREVIEW_BASE_URL', 'http://staging.mitx.mit.edu/courses/')
```

Comment:
change this to `https://example.com` or `https://www.sandbox.edx.org/. Also, the `courses` part will never change by configuration (it will always be courses no matter what server), so it isn't needed here and can be added in the URL construction.